src/pages/index.js

```diff
@@ -71,6 +60,32 @@ const cardSubmitBtn = editCardModal.querySelector(".modal__save-btn");
 const cardTemplate = document.querySelector("#card-template");
 const cardsList = document.querySelector(".cards__list");
 
+let selectedCard, selectedCardId;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6f4ae429-4ad9-43f6-abd7-d143d087e687",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getCardsAndUserInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.prepend(cardElement);
+    });
+
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+    document.querySelector(".profile__picture").src = userInfo.avatar;
```

Comment:
`document.querySelector(".profile__picture")`

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine